app/services/wds-on-site-notifications.js

```diff
@@ -80,12 +80,16 @@ export default Service.extend({
 		this.get('model').markAllAsRead();
 	},
 
-	markAsRead(notification) {
-		this.get('model').markAsRead(notification);
+	markAsRead(notification, willUnloadPage = false) {
+		return this.get('model').markAsRead(notification, willUnloadPage);
 	},
 
 	getUnreadCount() {
 		return this.get('model.unreadCount');
-	}
+	},
 
+	goToDestination(notification) {
+		// TODO check if it's a local url and use transitionTo or delegate to app
```

Comment:
what about this TODO?

Reply:
It's not a part of this ticket as it's already like that on production.
See https://wikia-inc.atlassian.net/browse/MAIN-16952